packages/target-nodejs-sdk/src/api/utils.js

```diff
@@ -0,0 +1,49 @@
+import { isDefined, isFunction, isUndefined } from "@adobe/target-tools";
+import { Messages } from "../messages";
+
+function withLabel(label, value) {
+  if (isUndefined(value)) {
+    return undefined;
+  }
+  return `\n${label}: ${value}`;
+}
+
+export function logApiResponse(
+  logger,
+  response,
+  decisioningMethod,
+  requestURL = undefined
+) {
+  if (isDefined(logger) && isFunction(logger.debug)) {
```

Comment:
Since this chunk is also duplicated in the other logging function we can extract a generic method here which would just get the arg array as input

Reply:
good idea